release/release.go

```diff
@@ -151,6 +151,38 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client, version string) (string, error) {
+	const (
+		owner string = "kubernetes"
+		repo  string = "kubernetes"
+	)
+
+	file, _, _, err := client.Repositories.GetContents(ctx, owner, repo, ".go-version", &github.RepositoryContentGetOptions{
+		Ref: version,
+	})
+
+	if err != nil {
+		switch err := err.(type) {
```

Comment:
Go provides helper functions in the stdlib for checking the error type. Can you update this to use `errors.Is` https://pkg.go.dev/errors#Is

Reply:
I got some errors using Is, so I changed to As passing a github error variable, also I added more error messages